### ISSUE TITLE
Question on the example: 

### ISSUE THREAD

**Issue by shersoni610** [openai/CLIP#380]
Hello,

I see the following example on the page: 

Qs: (1) we are calculate image and text features, but they are not being used in the code.
       (2) Are the arguments to the model (image,text) or (image_feature, text_features)

Thanks 

mport torch
import clip
from PIL import Image

device = "cuda" if torch.cuda.is_available() else "cpu"
model, preprocess = clip.load("ViT-B/32", device=device)

image = preprocess(Image.open("CLIP.png")).unsqueeze(0).to(device)
text = clip.tokenize(["a diagram", "a dog", "a cat"]).to(device)

with torch.no_grad():
    image_features = model.encode_image(image)
    text_features = model.encode_text(text)
    
    logits_per_image, logits_per_text = model(image, text)
    probs = logits_per_image.softmax(dim=-1).cpu().numpy()

print("Label probs:", probs)  # prints: [[0.9927937  0.00421068 0.00299572]]

**Comment by jongwook** [comment]
Those are intentional. Please see:

* https://github.com/openai/CLIP/pull/368#issuecomment-1626973523
